### ISSUE TITLE
Change tabled data into list output for blind players

### ISSUE THREAD

**Issue by luciensadi** [luciensadi/AwakeMUD#287]
SCORE presents a table, but should be in list form if TOGGLE SCREENREADER is enabled.

COMMANDS presents a table.

PGROUP presents a table.

**Comment by luciensadi** [comment]
POOLS = table

**Comment by luciensadi** [comment]
/u/Hades_Kane:
```
We still have a long way to go before I'd consider us truly screen reader friendly... things like breaking up our score sheet and similar things like that are probably where we need the greatest amount of work (I'm thinking of being able to use the score command to filter as broadly or specifically as possible... such as "score stats" would return ALL stats, defense, critical hit rate, etc. while "score vitals" might only return Power, Vitality, Agility, and Willpower... "score power" would just return power, "score defense" would return physical and magical defense, etc.)
```

**Comment by luciensadi** [comment]
Changed SCORE from ascii table to line-by-line readout. Still need to add breakout commands.

**Comment by luciensadi** [comment]
SCORE breakout commands complete.

**Comment by luciensadi** [comment]
COMMANDS now prints a giant list in screenreader mode. To assist in reading it, I've changed the behavior of COMMANDS [argument]; previously it would show you the commands for the target character you entered as an argument, but now it shows you all commands beginning with the prefix you enter.

**Comment by luciensadi** [comment]
Split up POOLS.

**Comment by luciensadi** [comment]
Split up PGROUP. This round of table-splitting is complete; will file more issues if I come across others.
